### DISCUSSION TITLE
可以不可以将 Mapbox token 写到环境变量中？

### DISCUSSION THREAD

**Discussion by 85Ryan** [yihong0618/running_page#573] [General]
想使用自己的 Mapbox token，但不想明文展示，请问可不可以将 Token 写到环境变量中？

谢谢！

**Comment by yihong0618** [comment]
应该可以，我们稍后支持下

**Comment by changchiyou** [comment]
@yihong0618 沒注意到已經有既有的相關討論串，還另外新開了個 Issue，不好意思

@85Ryan 可以參考 https://github.com/yihong0618/running_page/issues/643 ，懶人包：設置成環境變量、Github Secret 不會對資安有太大幫助，建議從 Mapbox 的 Dashboard 設置 URL Restriction。

**Reply by yihong0618** [reply]
谢谢。你的方案更好
